fixedwidth/fixedwidth.py

```diff
@@ -187,7 +199,13 @@ def _build_line(self):
         for field_name in [x[1] for x in self.ordered_fields]:
 
             if field_name in self.data:
-                datum = str(self.data[field_name])
+                type = self.config[field_name]['type']
```

Comment:
`type` is a reserved word in Python, so this should be changed. Since it doesn't look like it's being used more than once, maybe just use `self.config[field_name]['type']` in that one place.

Reply:
hey, i will remove it and use the format functions provided in https://github.com/ShawnMilo/fixedwidth/pull/12 as soon as it is merged